test/test_facebook.py

```diff
@@ -130,5 +130,17 @@ def test_auth_url(self):
         self.assertEqual(actual_query, expected_query)
 
 
+class TestGetObject(FacebookTestCase):
+    def test_get_object(self):
+        # The value we are passing as 'fields' is valid only in 2.1+.
+        graph = facebook.GraphAPI(access_token=facebook.get_app_access_token(
```

Comment:
`facebook.get_app_access_token` is deprecated and might be removed in the near future. Can we use the `GraphAPI().get_app_access_token` here?